packages/collector/test/tracing/messaging/amqp/test.js

```diff
@@ -24,198 +24,202 @@ let consumerControls;
 
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
 
-mochaSuiteFn('tracing/amqp', function () {
-  this.timeout(config.getTestTimeout());
+['latest', '0.8.0'].forEach(version => {
+  mochaSuiteFn(`tracing/amqp: ${version}`, function () {
+    this.timeout(config.getTestTimeout());
 
-  globalAgent.setUpCleanUpHooks();
+    globalAgent.setUpCleanUpHooks();
 
-  publisherControls = require('./publisherControls');
-  consumerControls = require('./consumerControls');
+    publisherControls = require('./publisherControls');
+    consumerControls = require('./consumerControls');
 
-  ['Promises', 'Callbacks'].forEach(apiType => {
-    describe(apiType, function () {
-      registerTests.call(this, apiType);
+    ['Promises', 'Callbacks'].forEach(apiType => {
+      describe(apiType, function () {
+        registerTests.call(this, apiType, version);
```

Comment:
@kirrg001  I removed the `version` parameter here in the latest commit, there is no `versions` parameter in the function definition of `registerTests`. Both the call and the function definition are in the scope of the outermost `forEach` so there is no need to pass it along as a parameter.